### DISCUSSION TITLE
Just a random question...

### DISCUSSION THREAD

**Discussion by chibibaka** [minegasm/minegasm-legacy#10] [General]
Hi,
I want to ask if it is possible to translate the code to run on Spigot servers?
Is it even possible to make such a plugin for Spigot?

I'm asking because we already have a server that uses Spigot and plugins. Therefore we can't really change to a forge Server.
It would be enough if I had the config options and the event handler onAttack and maybe onHurt, onCriticalHit.

Hope you have a nice day :)
Best regards

**Comment by RSwoop** [comment]
I'm still pretty new to Minecraft mod development so the real answer is I don't know. In theory, however, I don't see any reasons why it shouldn't be possible when sufficient effort is spent to do it. Personally, I'm afraid I don't have time to do it anytime soon given my limited time to work on this side project.

**Comment by RSwoop** [comment]
I was just informed that there is this new project for Spigot!
https://github.com/Cyloci/ButtplugMc
